src/helper.unit.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  it,
+  type Mock,
+  vi,
+} from 'vitest'
+import { checkPackageUpdates } from './helpers.js'
+
+const latestVersion = '2.5.6'
+
+describe('helpers', () => {
+  describe('checkPackageUpdates', () => {
+    beforeEach(() => {
+      vi.spyOn(global, 'fetch').mockResolvedValue({
+        json: () => Promise.resolve({ version: latestVersion }),
+      } as Response)
+
+      vi.spyOn(console, 'warn').mockImplementation(() => {})
+    })
```

Comment:
Should we move this to msw as well? 😄

Reply:
Can do

Reply:
Thinking about it the latest version endpoint isn't under our control and isn't so likely to change - I think mocking with fetch is fine